nnstreamer_example/tizen_app_build_example/CMakeLists.txt

```diff
@@ -6,7 +6,8 @@ OPTION(INSTALL_EXAMPLE_APP "Install example applications" ON)
 
 SET(PREFIX ${CMAKE_INSTALL_PREFIX})
 SET(EXEC_PREFIX "${PREFIX}/bin")
-SET(INCLUDEDIR "${PREFIX}/include/${PROJECT_NAME}")
+SET(INCLUDEDIR "${PREFIX}/include/nnstreamer")
```

Comment:
Is there any special reason to use a fixed string "nnstreamer" instead of "${PROJECT_NAME}"?

Reply:
nnstreamer installs h files to ${PREFIX}/include/nnstreamer

Reply:
I understood. Since this application is a sub-project of main CMake in project root directory so it looks OK.